packages/craco/README.md

```diff
@@ -6,43 +6,47 @@ Get all the benefits of create-react-app **and** customization without using 'ej
 
-**Support**
+## Support
 
-- CRA 3.*
+- Create React App (CRA) 3.*
 - Yarn
 - Yarn Workspace
 - NPM
 - Lerna (with or without hoisting)
 - Custom `react-scripts` version
 
-**Guide**
-
-- [Custom location for craco.config.js](#custom-location-for-cracoconfigjs) - Customize the location of your craco.config.js file.
-- [Configuration Overview](#configuration-overview) - Quickly see how you can configure your CRA installation with this plugin.
+## Documentation
+- [Preface](#preface)
+- [Installation](#installation)
+- [CRACO Configuration](#craco-configuration)
+  - [Sample CRACO Configuration File](#sample-craco-configuration-file)
+  - [The Mode Property](#the-mode-property)
+  - [Setting a Custom Location for `craco.config.js`](#setting-a-custom-location-for-cracoconfigjs)
+- [Extra Help for React and Webpack Toolchain Beginners](#extra-help-for-react-and-webpack-toolchain-beginners)
+  - [Ejecting CRA to Learn](#ejecting-cra-to-learn)
+  - [Notes on CRA Configurations and Problem Solving](#notes-on-cra-configurations-and-problem-solving)
+  - [Direct Versus Functional Config Definitions](#direct-object-literal-versus-functional-config-definitions)
 - [API](#api) - Have a look at CRACO APIs for Jest and Webpack.
-- [Recipes](https://github.com/sharegate/craco/tree/master/recipes) – Short recipes for common use cases.
-- [Available Plugins](https://github.com/sharegate/craco#community-maintained-plugins) - Plugins maintained by the community.
-- [Develop a Plugin](#develop-a-plugin) - How to develop a plugin for `craco`.
+- [Recipes and Plugins](#recipes-and-plugins)
+  - [Recipes](https://github.com/sharegate/craco/tree/master/recipes) – Short recipes for common use cases.
+  - [Available Plugins](https://github.com/sharegate/craco#community-maintained-plugins) - Plugins maintained by the community.
+  - [Developing a Plugin](#develop-a-plugin) - How to develop a plugin for `craco`.
 - [Changelog](https://github.com/sharegate/craco/tree/master/changelog) - List of major changes
 
-**Acknowledgements**
-
+## Preface
+### Acknowledgements
 We are grateful to [@timarney](https://github.com/timarney) the creator of [react-app-rewired](https://github.com/timarney/react-app-rewired) for his original idea.
 
-Also, please note that the configuration style of this plugin has been greatly influenced by the way [Vue CLI](https://cli.vuejs.org/guide/) does it.
-
-**Please Note**
+The configuration style of this plugin has been greatly influenced by [Vue CLI](https://cli.vuejs.org/guide/).
 
+### Fair Warning
 By doing this you're breaking the ["guarantees"](https://github.com/facebookincubator/create-react-app/issues/99#issuecomment-234657710) that CRA provides. That is to say you now "own" the configs. **No support** will be provided. Proceed with caution.
 
-## You updated craco and everything falls apart
-
+### Problems after updating CRACO?
```

Comment:
I think we should remove any information about this changelog. It is not used and release informations should be added well to the officiel release log.

Reply:
Removed, but left note about logging an issue.